docker/wait-for.sh

```diff
@@ -20,6 +20,10 @@ echoerr() {
   if [ "$QUIET" -ne 1 ]; then printf "%s\n" "$*" 1>&2; fi
```

Comment:
are we needed in strange license above?

Reply:
>The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

It does not bother us and the license require it, so why not leave it?